.gitignore

```diff
@@ -7,3 +7,4 @@ src/glyphs.h
 *.pyc
 .idea
 .vscode
+build
```

Comment:
You should be able to remove src/glyphs.h and .c from the .gitignore as it is under build directory now